### PR TITLE
Add 2019-07-03 infrastructure-operations meeting notes

### PR DESCRIPTION
**Pull request by ASoTNetworks**

### CODE REVIEW

File: 2019-07-03-infrastructure-operations-meeting.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# 2019-07-03 Toronto Workers Co-op: All-Hands Meeting
+
+<sup>[from template][template]</sup>
+
+[:date: Calendar][calendar] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive] | [:notebook: Meeting Index][meetings]
+
+Time: Wed, June 12, 2019 @ 17.00-18.00 ET
```
Comment (patcon) [inline 2019-07-03-infrastructure-operations-meeting.md]:
```suggestion
Time: Wed, July 3, 2019 @ 17.00-18.00 ET    
```

File: 2019-07-03-infrastructure-operations-meeting.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - dc: is that poll the final dec'n? I wasn't sure if we needed more documenting based on previous call
+    - patcon: Feeling blocked, want to start using sometime. yurko feels this pain
+    - dc: just wanted to make sure I understood this process
+- Next steps:
+    - Poll is going to close
+    - Poll for next all hands meeting
+- Values and our tools - parked until we had more time
+    - #todo dc to do some set up for that convo later
+- Accounting/billing
+    - Do we need to have accounting software set up RIGHT NOW
```
Comment (patcon) [inline 2019-07-03-infrastructure-operations-meeting.md]: Was the answer "yes, this is urgent" here? I recall it being quite urgent.

File: 2019-07-03-infrastructure-operations-meeting.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    - More complicated things: auditing, HSTs, payroll
+    - #todo patcon: ask cotech folks and briarwood co-op how they do their books and invoicing
+    - #todo dc: call kenzie; co-op creator; OCA?
```
Comment (patcon) [inline 2019-07-03-infrastructure-operations-meeting.md]: Are these the separate todos? Not sure I understand the actions from the shorthand :)

```
- #todo dc: call Kenzie Love at OCA
- #todo dc: look for support in [Co-op Creator](https://coopcreator.ca/)

File: 2019-07-03-infrastructure-operations-meeting.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    - Do we need to have accounting software set up RIGHT NOW
+    - More complicated things: auditing, HSTs, payroll
+    - #todo patcon: ask cotech folks and briarwood co-op how they do their books and invoicing
+    - #todo dc: call kenzie; co-op creator; OCA? 
+
+## Follow Up
+
+Next meeting? 
+2 weeks? dc and yurko can't make it
+3 weeks? sounds good -- Jul 24 (All Hands)
```
Comment (patcon) [inline 2019-07-03-infrastructure-operations-meeting.md]: My understanding was that we weren't talking about the all hands slot? So is the #todo this:

```
- #todo patcon: schedule infra call in week of July 24
```

File: 2019-07-03-infrastructure-operations-meeting.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+### Actionable Items
+
+- [ ] dc: record voicemail for Hypha
+- [ ] patcon: non-urgent task: to switch to using link shortner to version controlled csv in github over gdocs
+- [ ] patcon/dc: add link shortner to the handbook
+- [ ] dc: do some set up for values and our tools talk
+- [ ] patcon: ask cotech folks and briarwood co-op how they do their book
+- [ ] dc: call kenzie; co-op creator; OCA? 
+- [ ] patcon: set up July 24 meeting + biweekly infra meetings
+- [ ] elon: add notes to gh
```
Comment (patcon) [inline 2019-07-03-infrastructure-operations-meeting.md]:
```suggestion
- [x] elon: add notes to gh
```

🙌

File: 2019-07-03-infrastructure-operations-meeting.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+- Checked-in on action items
+- Task Tracker
+    - Loomio polls 2 days for items
+        - all "infra folks" have responded to two questions
+        - internal: GitHub projects still winning
+    - dc: is that poll the final dec'n? I wasn't sure if we needed more documenting based on previous call
+    - patcon: Feeling blocked, want to start using sometime. yurko feels this pain
+    - dc: just wanted to make sure I understood this process
+- Next steps:
+    - Poll is going to close
```
Comment (patcon) [inline 2019-07-03-infrastructure-operations-meeting.md]:
```suggestion
    - ranking poll is going to close over weekend
```

File: 2019-07-03-infrastructure-operations-meeting.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    - Poll for next all hands meeting
```
Comment (patcon) [inline 2019-07-03-infrastructure-operations-meeting.md]:
```suggestion
    - Poll to make use of front-runner will be created and close by next all hands meeting in coming week.
```

File: 2019-07-03-infrastructure-operations-meeting.md (hunk 4 above)
Comment (patcon) [reply 2019-07-03-infrastructure-operations-meeting.md]:
```suggestion
3 weeks? yurko away (until jul 26). sounds good -- Jul 24 (All Hands)
```

File: 2019-07-03-infrastructure-operations-meeting.md (hunk 3 above)
Comment (dcwalk) [reply 2019-07-03-infrastructure-operations-meeting.md]: I reached out to those two, I didn't do it the day you've suggested so I'll propose a different wording if you want

Comment (patcon) [reply 2019-07-03-infrastructure-operations-meeting.md]: No it's ok, just didn't understand the actions when reviewing :) nbd

**Pull request merged**